### ISSUE TITLE
java -cp doesn't work

### ISSUE THREAD

**Issue by scotbellamy** [dthree/cash#104]
When trying to execute a java -cp command in the Cash shell, -cp isn't interpreted as a proper java command line argument and results in "Invalid option: 'c'. Showing Help: ...".  Would love to stay in the shell for all of my development work.

**Comment by nfischer** [comment]
Yeah, that sounds like it's the options parser. If we're parsing options for external commands, that's probably a bad approach, since some commands parse options in different ways.

If someone wants to take a look at this, I'll review a PR
